packages/terra-application-name/tests/nightwatch/DefaultApplicationName.jsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
```

Comment:
Move to /terra-applicaiton-name/examples/test-examples to match new site changes